dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -195,49 +200,49 @@ public String toJson(Contentlet con) throws DotMappingException {
 
-			m.put("title", con.getTitle());
-			m.put("structureName", st.getVelocityVarName()); // marked for DEPRECATION
-			m.put("contentType", st.getVelocityVarName());
-            m.put("structureType", st.getStructureType() + ""); // marked for DEPRECATION
-            m.put("baseType", st.getStructureType() + "");
-            m.put("type", "content");
-            m.put("inode", con.getInode());
-            m.put("modDate", datetimeFormat.format(con.getModDate()));
-            m.put("owner", con.getOwner()==null ? "0" : con.getOwner());
-            m.put("modUser", con.getModUser());
-            m.put("live", Boolean.toString(con.isLive()));
-            m.put("working", Boolean.toString(con.isWorking()));
-            m.put("locked", Boolean.toString(con.isLocked()));
-            m.put("deleted", Boolean.toString(con.isArchived()));
-            m.put("languageId", Long.toString(con.getLanguageId()));
-            m.put("identifier", ident.getId());
-            m.put("conHost", ident.getHostId());
-            m.put("conFolder", conFolder!=null && InodeUtils.isSet(conFolder.getInode()) ? conFolder.getInode() : con.getFolder());
-            m.put("parentPath", ident.getParentPath());
-            m.put("path", ident.getPath());
+			contentletMap.put(ESMappingConstants.TITLE, con.getTitle());
+			contentletMap.put(ESMappingConstants.STRUCTURE_NAME, st.getVelocityVarName()); // marked for DEPRECATION
+			contentletMap.put(ESMappingConstants.CONTENT_TYPE, st.getVelocityVarName());
+            contentletMap.put(ESMappingConstants.STRUCTURE_TYPE, st.getStructureType() + ""); // marked for DEPRECATION
+            contentletMap.put(ESMappingConstants.BASE_TYPE, st.getStructureType() + "");
+            contentletMap.put(ESMappingConstants.TYPE, ESMappingConstants.CONTENT);
+            contentletMap.put(ESMappingConstants.INODE, con.getInode());
+            contentletMap.put(ESMappingConstants.MOD_DATE, datetimeFormat.format(con.getModDate()));
+            contentletMap.put(ESMappingConstants.OWNER, con.getOwner()==null ? "0" : con.getOwner());
+            contentletMap.put(ESMappingConstants.MOD_USER, con.getModUser());
+            contentletMap.put(ESMappingConstants.LIVE, Boolean.toString(con.isLive()));
+            contentletMap.put(ESMappingConstants.WORKING, Boolean.toString(con.isWorking()));
+            contentletMap.put(ESMappingConstants.LOCKED, Boolean.toString(con.isLocked()));
+            contentletMap.put(ESMappingConstants.DELETED, Boolean.toString(con.isArchived()));
+            contentletMap.put(ESMappingConstants.LANGUAGE_ID, Long.toString(con.getLanguageId()));
+            contentletMap.put(ESMappingConstants.IDENTIFIER, ident.getId());
+            contentletMap.put(ESMappingConstants.CONTENTLET_HOST, ident.getHostId());
+            contentletMap.put(ESMappingConstants.CONTENTLET_FOLER, conFolder!=null && InodeUtils.isSet(conFolder.getInode()) ? conFolder.getInode() : con.getFolder());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 167 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)